src/Worker/Core/Logs.cs

```diff
@@ -9,15 +9,35 @@ namespace Microsoft.DurableTask
     /// <summary>
     /// Log messages.
     /// </summary>
-    /// <remarks>
-    /// NOTE: Trying to make logs consistent with https://github.com/Azure/durabletask/blob/main/src/DurableTask.Core/Logging/LogEvents.cs.
-    /// </remarks>
     static partial class Logs
     {
+        /// <summary>
+        /// The common category name for worker logs.
+        /// </summary>
+        internal const string WorkerCategoryName = "Microsoft.DurableTask.Worker";
+
         [LoggerMessage(EventId = 15, Level = LogLevel.Error, Message = "Unhandled exception in entity operation {entityInstanceId}/{operationName}.")]
         public static partial void OperationError(this ILogger logger, Exception ex, EntityInstanceId entityInstanceId, string operationName);
 
         [LoggerMessage(EventId = 55, Level = LogLevel.Information, Message = "{instanceId}: Evaluating custom retry handler for failed '{name}' task. Attempt = {attempt}.")]
         public static partial void RetryingTask(this ILogger logger, string instanceId, string name, int attempt);
+
+        [LoggerMessage(EventId = 600, Level = LogLevel.Information, Message = "'{Name}' orchestration with ID '{InstanceId}' started.")]
+        public static partial void OrchestrationStarted(this ILogger logger, string instanceId, string name);
+
+        [LoggerMessage(EventId = 601, Level = LogLevel.Information, Message = "'{Name}' orchestration with ID '{InstanceId}' completed.")]
+        public static partial void OrchestrationCompleted(this ILogger logger, string instanceId, string name);
+
+        [LoggerMessage(EventId = 602, Level = LogLevel.Information, Message = "'{Name}' orchestration with ID '{InstanceId}' failed.")]
```

Comment:
Should this be error?

Reply:
I go back and forth on this one. It might be an error from the app's perspective, but it's not an error from the framework's perspective.